client/telecomUniverseComponents/phone/number/phone-number.directive.js

```diff
@@ -1,4 +1,6 @@
-angular.module('managerApp').directive('phoneNumber', ($q, OvhApiValidateAapi, TelephonyMediator) => ({
+import _ from 'lodash';
+
+export default /* @ngInject */ ($q, OvhApiValidateAapi, TelephonyMediator) => ({
```

Comment:
reminder: it should be update as soon a we migrate the `TelephonyMediator` component.